docs/README.md

```diff
@@ -17,35 +17,370 @@ with a simple and easy to use interface for working with files stored in the Alf
+
+- Document View: PDFs are displayed in the application File Viewer, for other document types (DOCX etc) then a PDF rendition is automatically retrieved.
+- Image View: JPEG, PNG, GIF, BMP and SVG images are natively displayed in the application File Viewer.
+- Media View: MP4, MP3, WAV, OGG and WEBM files are played natively application File Viewer. The File Viewer will download, by default, 50MB of the content at a time to ensure a smooth playback of the content.
+- Text View: TXT, XML, JS, HTML, JSON and TS files are natively displayed as text in the application File Viewer.
+
+#### Thumbnails side pane
+
+The Document View includes a thumbnails pane which can be activated by a button in the Viewer Actions toolbar. Clicking on a thumbnail will take a user directly to the selected page and as users scroll through a document the current page is highlighted in the pane.
+
+#### Viewer Controls
+
+At the bottom of the content the Viewer Controls allow users to interact with the content in various ways; the actions available are dependant on the type of content being displayed.
+
+- Document View:
+   - Activate/Deactivate thumbnails pane
+   - Previous/Next page
+   - Jump to page number
+   - Zoom in/out
+   - Fit to page
+- Image View:
+   - Zoom in/out
+   - Rotate left/right (does not alter content in the repository)
+   - Reset image
+- Media View:
+   - Play/pause
+   - Timeline position
+   - Audio mute/unmute
+   - Audio volume
+   - Full screen
+
+### Info Drawer
+
+The Info Drawer displays node information in the right sidebar panel. It is created by using the [InfoDrawerComponent](https://alfresco.github.io/adf-component-catalog/components/InfoDrawerComponent.html). This info is available for both folder and file nodes.
+
+Currently, there are 2 tabs available: Properties and Versions.
+
+#### Properties tab
+
+The Properties tab displays the node's metadata info by using the [ContentMetadataCardComponent](https://alfresco.github.io/adf-component-catalog/components/ContentMetadataCardComponent.html).
+
+![](images/content-metadata.png)
+
+For more information, please check also the ADF's [ContentMetadataComponent](https://alfresco.github.io/adf-component-catalog/components/ContentMetadataComponent.html).
+
+#### Versions tab
+
+The Versions tab displays info about the node's versions and allows users to [manage versions](/version-manager), according to their permissions. Only the file nodes have version data available.
+
+![](images/version-manager-tab.png)
+
+It uses the [VersionManagerComponent](https://alfresco.github.io/adf-component-catalog/components/VersionManagerComponent.html) from ADF framework.
+
+Managing versions of a file can be possible also by accessing the 'Manage Versions' option from the 'More actions' menu. For more info on manage versions, please check the [version manager](/version-manager) page.
```

Comment:
the link to version manager is broken also

Reply:
thanks, fixed